packages/notifier/src/types-ambient.js

```diff
@@ -160,6 +160,8 @@
  *
  * @property {DurablePublishKitValueDurability} valueDurability
  *
+ * @property {import('@agoric/internal/src/callback.js').Callback<(publicationRecordP: Promise<PublicationRecord<*>>) => void>} [onUpdate]
```

Comment:
I like this type.  Since it's reused in this package, would it be worth a `@typedef {...} PublishKitOnUpdate` somewhere here?

Reply:
Sure; done.